### DISCUSSION TITLE
Airbyte Temporal container can't start, `dynamicconfig/development.yaml: no such file`

### DISCUSSION THREAD

**Discussion by awahuang** [airbytehq/airbyte#28590] [Connector Questions] [answered]
container start log  message:
Unable to create dynamic config client. Error: unable to validate dynamic config: dynamic config: config/dynamicconfig/development.yaml: stat config/dynamicconfig/development.yaml: no such file or directory

i fllow the Deploy Airbyte step installation
Thanks
Version:0.5.10

**Comment by marcosmarxm** [comment, marked as answer]
Hello 👋  the issue was fixed. The cause was last week Airbyte upgrade Temporal version to use a more recent version of their service and also Airbyte started to use a `dynamicconfig` file. The file was added in the Docker compose but the .run-platform.sh script wasn’t updated.

**What you need to do is update the `run-ab-platform.sh` script which now includes the new file.**

More info https://github.com/airbytehq/airbyte/issues/28629

**Reply by awahuang** [reply]
Great thanks
